src/descriptor/policy.rs

```diff
@@ -139,6 +170,9 @@ pub enum SatisfiableItem {
         keys: Vec<PkOrF>,
         /// The required threshold count
         threshold: usize,
+        /// The script type used to encode
+        #[serde(skip_serializing_if = "Option::is_none")]
+        script_type: Option<MultisigScriptType>,
```

Comment:
I am feeling like we should be able to set the `ScriptType` in the `Policy` structure itself. Not just only multisigs.. And this should not be inside `SatisfiableItem`..